core/src/main/scala/chisel3/Mem.scala

```diff
@@ -122,9 +171,38 @@ sealed abstract class MemBase[T <: Data](val t: T, val length: BigInt)
   )(
     implicit evidence: T <:< Vec[_],
     compileOptions:    CompileOptions
+  ): Unit =
+    masked_write_impl(idx, data, mask, Builder.forcedClock, true)
+
+  def write(
```

Comment:
Please add ScalaDoc like the `write` method above does (for every added public, non-do_ method). You can basically copy the ScalaDoc but add something about the `clock` parameter and what it's for.